src/components/Trade/hooks/useTradeAmounts.tsx

```diff
@@ -175,8 +175,8 @@ export const useTradeAmounts = () => {
       const tradeQuoteArgs = await getTradeQuoteArgs({
         buyAsset,
         sellAsset,
-        sellAccountType: sellAccountMetadata.accountType,
-        sellAccountBip44Params: sellAccountMetadata.bip44Params,
+        sellAccountType: sellAccountMetadata?.accountType,
```

Comment:
we should early return/throw if we don't have `sellAccountMetadata`, opt-chaining here is a bug

Reply:
4cf9ce7b81fc34ba57920d68f0e74faae27eeadc